lib/src/install.rs

```diff
@@ -638,7 +644,11 @@ async fn install_container(
 ) -> Result<(ostree::Deployment, InstallAleph)> {
     let sepolicy = state.load_policy()?;
     let sepolicy = sepolicy.as_ref();
-    let stateroot = STATEROOT_DEFAULT;
+    let stateroot = state
+        .config_opts
+        .stateroot
+        .as_deref()
+        .unwrap_or(ostree_ext::container::deploy::STATEROOT_DEFAULT);
```

Comment:
Worth reviewing, I wasn't sure whether to duplicate this or just mutate state with the resolved stateroot

Reply:
Hmm...perhaps slightly cleaner as a `fn stateroot(&self) -> &str` on `State`?